spec/formatting.md

```diff
@@ -115,7 +115,8 @@ and a _function_, _private-use_, or _reserved_ _annotation_,
 the resolved value of the _expression_ is determined as follows:
 
 If the _expression_ contains a _reserved_ _annotation_,
-an `Unsupported Expression` error is emitted and a fallback value is used as its value.
+an Unsupported Expression error is emitted and
```

Comment:
```suggestion
an `Unsupported Expression` error is emitted and
```

(missing the quoting of the error type?)

Reply:
No, updating it to match the prevailing convention—all of the other error types ("Unresolved Variable error", "Unknown Function error", etc.) in https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md are unformatted.